lib/utils.js

```diff
@@ -283,6 +283,25 @@ const utils = {
         const name = stdout.replace('\n', '')
         return exec(`git tag v${name} -a -m "Generated tag from CI build ${config.buildNumber}"`)
       })
+  },
+
+  /**
+   * Create a release branch if there is a major bump
+   * @param {PrInfo} info - the pr info
+   * @returns {PrInfo} the pr info object passed in
+   */
+  createReleaseBranch (info) {
+    if (info.scope === 'major') {
+      // use info.version
+      exec(`git checkout -b  ${info.version}`, function (error, stdout, stderr) {
+        if (error !== null) {
+          console.log(`exec error: ${error}`)
+        } else {
+          exec(`git push origin ${info.version}`)
+        }
+      })
+    }
+    return info
```

Comment:
There's a couple problems I see here:

 1. `createReleaseBranch()` doesn't actually always create a release branch, so the function might be better named `maybeCreateReleaseBranch()` or something to that effect. 
 1. As mentioned in the general comment, you're creating a branch off of the new change, what you really want is a branch created from the most recent tag *before* this tag gets made. I can incorporate this feature into the bitbucket branch I'm working on.